beluga/CMakeLists.txt

```diff
@@ -48,6 +48,13 @@ find_package(range-v3 REQUIRED)
 find_package(Sophus REQUIRED)
 find_package(TBB REQUIRED)
 
+list(APPEND CMAKE_MODULE_PATH "/usr/lib/x86_64-linux-gnu/cmake/OpenVDB/")
+list(APPEND CMAKE_MODULE_PATH "/usr/lib/aarch64-linux-gnu/cmake/OpenVDB")
```

Comment:
@pvela2017 instead of blanket adding these paths, consider:

- adding a `USE_OPENVDB` [option](https://cmake.org/cmake/help/latest/command/option.html) that defaults to `false`
- using [`find_file`](https://cmake.org/cmake/help/latest/command/find_file.html) to look for the `FindOpenVDB.cmake` path under `/usr/lib` (and `/usr/local/lib` perhaps)
- [`set`](https://cmake.org/cmake/help/latest/command/set.html#set-cache-entry)ting an `OPENVDB_CMAKE_MODULE_PATH` using the parent path of the module path found, allowing the user to override it

You can skip almost everything if `USE_OPENVDB` is false.

Reply:
Updated accordingly in bef40ad2dd85beba55969b16c62adab2f0f94e79